kythe/java/com/google/devtools/kythe/extractors/java/JavaCompilationUnitExtractor.java

```diff
@@ -703,110 +712,39 @@ private AnalysisResults runJavaAnalysisToExtractCompilationDetails(
       throws ExtractionException {
 
     AnalysisResults results = new AnalysisResults();
+    DiagnosticCollector<JavaFileObject> diagnosticCollector = new DiagnosticCollector<>();
+    CompilationUnitCollector compilationCollector = new CompilationUnitCollector();
 
-    // We will initialize and run the Javac compiler to detect which dependencies
-    // the current compilation has.
-    JavaCompiler compiler = findJavaCompiler();
-    if (compiler == null) {
```

Comment:
Why was this removed?

Reply:
It was not; it was moved into `getJavaCompiler`